A.D./src/main/java/com/adoptdifferently/model/User.java

```diff
@@ -23,7 +23,7 @@ public class User {
         this.country = country;
         this.city = city;
         this.postcode = postcode;
-        this.userId = userId;
+        this.userId = 1;
```

Comment:
Why are you only setting user to the number one

Reply:
My bad I meant to set that as a parameter instead of hardcoding it. I started working on this last night so I don't really know what sleep-deprived me was thinking of.

Reply:
I changed the issue but I most likely gotta modify the test. Ima do that later.